Documentation/ReleaseNotes.md

```diff
@@ -1,5 +1,9 @@
 # <a id="release-notes"></a>Release Notes
 
+## Version 3.7 (November 2023)
```

Comment:
a Tung. a think we don't suggest and also want user don't using auto refreshing headset scan. 
we want to change user behavior that click refresh button each time want to scan a device. 
so what we can implement is: 
- In Unity plugin: implement 1 function allow to call to refresh API. 
- In Unity example: we implement 1 more refresh button and ask user click it each time want to scan a device.
It also show the message Scan done to let them know when need to click next time. 
- Auto refresh is ONLY in our Emotiv applications and it is in UI layer. not Unity plugin layer. 

what do you think @thamha-emotiv ?